kpi/tests/api/v2/test_api_assets.py

```diff
@@ -601,17 +601,19 @@ def setUp(self):
         self.client.login(username='someuser', password='someuser')
         self.current_username = 'someuser'
         self.asset = Asset.objects.filter(owner__username='someuser').first()
-        self.list_url = reverse(
-            self._get_endpoint('asset-file-list'), args=[self.asset.uid]
-        )
+        self.list_url = reverse(self._get_endpoint('asset-file-list'), args=[self.asset.uid])
```

Comment:
🤔  I'm wondering why `Black` did not suggest the format I used over yours. This line seems to go beyond the 80 characters limit.

Reply:
I was mistaken to include this change in this PR :frowning: I don't think I'll make a separate commit or PR to fix it, though. It'll just be the next person in here who fixes my sloppiness :smirk:

Reply:
I thought exactly the same :-)